app/remote_cluster.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package app
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+func (a *App) AddRemoteCluster(rc *model.RemoteCluster) (*model.RemoteCluster, error) {
+	return a.Srv().Store.RemoteCluster().Save(rc)
+}
+
+func (a *App) DeleteRemoteCluster(remoteClusterId string) (bool, error) {
+	return a.Srv().Store.RemoteCluster().Delete(remoteClusterId)
+}
+
+func (a *App) GetRemoteCluster(remoteClusterId string) (*model.RemoteCluster, error) {
+	return a.Srv().Store.RemoteCluster().Get(remoteClusterId)
+}
+
+func (a *App) GetAllRemoteClusters(incOffline bool) ([]*model.RemoteCluster, error) {
```

Comment:
nit: I'd prefer `includeOffline`.
```suggestion
func (a *App) GetAllRemoteClusters(incOffline bool) ([]*model.RemoteCluster, error) {
```

Reply:
Done.  Changed in a bunch of places to be consistent.